src/ipc/cc_version.c

```diff
@@ -0,0 +1,34 @@
+// SPDX-License-Identifier: BSD-3-Clause
+//
+// Copyright(c) 2016 Intel Corporation. All rights reserved.
+//
+// Author: Karol Trzcinski <karolx.trzcinski@linux.intel.com>
+
+#include <ipc/info.h>
+#include <sof/common.h>
+#include <sof/compiler_info.h>
+
+/* copy CC_NAME to const arrays during compilation time */
+#define CC_NAME_COPY(field) \
+	field = CC_NAME, \
+	field[ARRAY_SIZE(((struct sof_ipc_cc_version *)(0))->name) - 1] = 0
+
+/* copy CC_OPTIM to const arrays during compilation time */
+#define CC_OPTIM_COPY(field) \
+	field = CC_OPTIMIZE_FLAGS, \
+	field[ARRAY_SIZE(((struct sof_ipc_cc_version *)(0))->optim) - 1] = 0
+
+const struct sof_ipc_cc_version cc_version = {
+	.ext_hdr	= {
```

Comment:
There is tab before `=`, raw view confirms it: https://raw.githubusercontent.com/thesofproject/sof/e550a879cde3cc069b74f2e0f78d145f67516f61/src/ipc/cc_version.c

Reply:
replaced with single space